deployments/docker/dev/docker-compose-apple-silicon.yml

```diff
@@ -53,7 +53,7 @@ services:
   azurite:
     image: mcr.microsoft.com/azure-storage/azurite
     volumes:
-      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/azurite:/opt/azurite
+      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/azurite:/data
```

Comment:
The storage path had been changed for `azurite` ?

Reply:
![image](https://github.com/milvus-io/milvus/assets/21985684/a14b87e3-c618-49c7-a440-8d8ebcfc3df5)
link: https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub%2Cblob-storage

Reply:
The dev docker yaml is set incorrectly. If this `/opt/azurite` directory is mapped, the docker will not start.